bluepyemodel/emodel_pipeline/emodel_pipeline.py

```diff
@@ -122,12 +139,10 @@ def __init__(
             self.mapper = map
 
         endpoint = None
-
-        if data_access_point is not None and data_access_point != "local":
-            raise ValueError(
-                "Attempted to set a legacy variable. "
-                "This variable should not be modified in new code."
-            )
+        if nexus_endpoint == "prod":
```

Comment:
good point. I will update the comment to let the users know that they should provide the endpoint addresses. We can still allow users to specify `prod` and `staging` so that those who already use them don't have to change their pipeline.

[There](https://github.com/BlueBrain/BluePyEModel/blob/fd5f9d7c57f7719df3a05a78be1c453a9b7bb7c7/bluepyemodel/access_point/__init__.py#L89), `https://bbp.epfl.ch/nexus/v1` is the default value used only if the endpoint is not `prod` or `staging`. I could actually change it to staging `https://staging.nexus.ocp.bbp.epfl.ch/v1`